src/spotify_app.py

```diff
@@ -30,10 +30,14 @@
 sp_oauth = oauth2.SpotifyOAuth(SPOTIFY_CLIENT_ID, SPOTIFY_CLIENT_SECRET, SPOTIFY_REDIRECT_URI, scope=scope)
 
 
-@app.route("/index")
-def index():
-    return render_template('index.html', spotify_auth_url=get_spotify_oauth_url())
-
+@app.route("/login")
+def login():
+    return render_template('login.html')
+
+# TODO prevent refreshing on undefined route (like /page2) from borking app
```

Comment:
This is an issue

Reply:
Possible solution here https://stackoverflow.com/questions/30620276/flask-and-react-routing